From 388e1d898d0dd3879d6889ab2d65f007eff11bea Mon Sep 17 00:00:00 2001
From: SignKirigami <prcups@krgm.moe>
Date: Tue, 25 Aug 2026 11:02:52 +0800
Subject: [PATCH 6/6] AOSCOS: touchpad: hide unsupported X11 scroll speed row

Hide the entire form entry when scroll speed is unsupported so its label
and layout space are removed on X11.

Use gpt-5.6-sol to generate fix code.

Signed-off-by: SignKirigami <prcups@krgm.moe>
---
 kcms/touchpad/ui/main.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kcms/touchpad/ui/main.qml b/kcms/touchpad/ui/main.qml
index b6d083af31..8941417062 100644
--- a/kcms/touchpad/ui/main.qml
+++ b/kcms/touchpad/ui/main.qml
@@ -305,12 +305,12 @@ KCMUtils.SimpleKCM {
         Kirigami.FormGroup {
             // Scroll Speed aka scroll Factor
             Kirigami.FormEntry {
+                visible: root.device?.supportsScrollFactor ?? false
                 contentItem: GridLayout {
                     Kirigami.FormData.label: i18ndc("kcm_touchpad", "@label:slider", "Scrolling speed:")
                     Kirigami.FormData.buddyFor: scrollFactor
                     Layout.fillWidth: true
 
-                    visible: root.device?.supportsScrollFactor ?? false
                     columns: 3
 
                     QQC2.Slider {
-- 
2.55.0

